services/src/main/scala/cromwell/pipeline/service/UserService.scala

```diff
@@ -50,3 +55,8 @@ class UserService(userRepository: UserRepository)(implicit executionContext: Exe
       }
     } else Future.failed(new RuntimeException("new password incorrectly duplicated"))
 }
+
+object UserService {
+  def apply(userRepository: UserRepository)(implicit executionContext: ExecutionContext): UserService =
+    new UserService(userRepository)(executionContext)
+}
```

Comment:
This change is ok by itself, but it will make code style in `UserService` inconsistent with other services. We should either change all services to have an `apply` method, or don't change this one. Since we already have a task EPMLSTRCMW-241 where we create `apply` method for all services, I suggest you to just revert this change

Reply:
@myazinn done